client/v2/autocli/flag/field.go

```diff
@@ -2,14 +2,13 @@ package flag
 
 import (
 	"context"
-
 	autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+	"cosmossdk.io/client/v2/internal/util"
 	cosmos_proto "github.com/cosmos/cosmos-proto"
 	"github.com/spf13/pflag"
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/reflect/protoreflect"
-
-	"cosmossdk.io/client/v2/internal/util"
+	"strconv"
```

Comment:
Please move standard library imports to their own section, move non-cosmos-sdk imports to their own group then cosmos-sdk imports to the last block so
```go
import (
    "context"
    "strconv"

    "github.com/spf13/pflag"
    "google.golang.org/protobuf/proto"
    "google.golang.org/protobuf/reflect/protoreflect"

     autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
     "cosmossdk.io/client/v2/internal/util"
     cosmos_proto "github.com/cosmos/cosmos-proto"
)
```